src/index.ts

```diff
@@ -215,16 +216,18 @@ export function removeEmpty(obj: any): any {
  * and these fields are invalid when issuing an update, or PUT request, so they
  * must be removed before any update requests are sent.
  */
-export function cleanMetaData(obj: any): any {
-  /* eslint-disable */
-  const {
-    createdTime,
-    lastModifiedTime,
-    password,
-    status,
-    ...ret } = obj
-  /* eslint-enable */
-  return ret
+export interface fields {
+  [key:string]: string[]
+}
```

Comment:
Don't need this at all - see previous comment about `problemFields`...

Reply:
If the `problemFields` needs an interface defining type string used as an index, else this compiler error is thrown:

```
<html>TS7053: Element implicitly has an 'any' type because expression of type 'string' can't be used to index type '{ 'POST:usertransitions': string[]; }'.<br/>No index signature with a parameter of type 'string' was found on type '{ 'POST:usertransitions': string[]; }'.
```

Reply:
OK... as I'm reading the TypeScript docs, it _seems_ like it should be possible to say:

```typescript
  const problemFields: { [index: string]: string[] } = {
    'POST:usertransitions': ['createdTime', 'lastModifiedTime', 'password'],
  }
```

Does that work?